Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeNode.mm

```diff
@@ -238,4 +243,32 @@
     return [description stringByAppendingString:layerDescription];
 }
 
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
+void RemoteLayerTreeNode::setAcceleratedEffectsAndBaseValues(const WebCore::AcceleratedEffects& effects, const WebCore::AcceleratedEffectValues& baseValues, RemoteLayerTreeHost* host)
+{
+    ASSERT(isUIThread());
+    ASSERT(host);
+
+    if (m_effects)
+        host->animationEffectStackWasRemoved(*m_effects);
+
+    if (effects.isEmpty())
+        return;
+
+    m_effects = new RemoteAcceleratedEffectStack(layer().bounds, host->acceleratedTimelineTimeOrigin());
```

Comment:
I think this needs to have `adoptRef`.

Reply:
Yes, this should be behind `::create()` methods like RefPtr classes typically offer.